.gitignore

```diff
@@ -174,3 +174,6 @@ cython_debug/
 #  option (not recommended) you can uncomment the following to ignore the entire idea folder.
 .idea
 
+# For pyenv
+.python-version
```

Comment:
I use pyenv to automatically switch to this project's virtualenv. But figure we don't want that in the repo